content/dogepedia/articles/merchants-accepting-doge.md

```diff
@@ -5,4 +5,8 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Lists to shops and companies accepting Doge. Link to directories?
\ No newline at end of file
+At the time of writing, there is a rapidly expanding list of merchants who accept Dogecoin as a form of payment. Because of this, finding — and maintaining — an up-to-date directory of businesses that #AcceptDoge is no small feat!  
+
+Thanks to the overwhelming support from the Dogecoin community in encouraging — and then promoting — businesses that accept Dogecoin, we anticipate that the list of businesses that accept Dogecoin is only going to grow much larger and cover a variety of industries and sectors in the years to come. This is certainly the aim: the Dogecoin community is committed to seeing Dogecoin used as a viable alternative to traditional centralised fiat money.  
+
+In addition to large retailers who have signalled they will be accepting Dogecoin in the future, and some who do already, there are thousands of small businesses that accept Dogecoin throughout the world — and this grassroots adoption is crucial to Dogecoin’s success! If you have a business and are interested in accepting Dogecoin, learn more about how to do so [here] (/dogepedia/articles/how-can-my-business-accept-dogecoin). 
```

Comment:
```suggestion
In addition to large retailers who have signalled they will be accepting Dogecoin in the future, and some who do already, there are thousands of small businesses that accept Dogecoin throughout the world — and this grassroots adoption is crucial to Dogecoin’s success! If you have a business and are interested in accepting Dogecoin, learn more about how to do so [here] (/dogepedia/articles/how-can-my-business-accept-dogecoin). 

Web directories listing shops currently accepting Dogecoin can be found in the [Resources page](/resources/#shops).
```

Reply:
Seems great, maybe we can add this final line linking to the resources page, where we have listed some Shop directories?